templates/data/prow-periodics-data.yaml

```diff
@@ -206,11 +206,11 @@ templates:
                     owner: neighbors
                     description: "Autobump image versions in prow jobs"
                   cron: "45 * * * 1-5"
-                  slack_channel: kyma-prow-alerts
+                  slack_channel: kyma-neighbors-alerts
                   image: europe-docker.pkg.dev/kyma-project/prod/k8s-prow/generic-autobumper:v20231030-70c8254b
                   command: generic-autobumper
                   args:
-                    - --config=configs/autobump-config/test-infra-autobump-config.yaml
+                    - --config=prow/autobump-config/test-infra-autobump-config.yaml
```

Comment:
is this path correct after refactor of test infra?  I think it should be now here: https://github.com/kyma-project/test-infra/tree/main/configs/autobump-config

Reply:
corrected